src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -208,18 +203,6 @@ public override int Run(AnalyzeOptions analyzeOptions)
                 analyzeOptions.Level = new List<FailureLevel> { FailureLevel.Error, FailureLevel.Warning, FailureLevel.Note };
             }
```

Comment:
Doesn't this go away as well?
Your notes say you have dropped obsolete args like --verbose.

Reply:
@schlaman-ms Yes this needs to go away as well. We have a work item for removing it, but not sure if removing this would in scope for this PR.

Reply:
It's a good question. In general, if we're breaking command-line already, it might be good to batch this up (and remove the verbose thing as well). Let me see how costly it is. Certainly these changes need to be more explicit in the release notes, too.

Reply:
Turned out to be easy. Removed, updated release notes.